### PR TITLE
Loves 7.25 Rough Draft

### PR DESCRIPTION
**Pull request by lovesmuffins**

### CODE REVIEW

File: game/scripts/npc/abilities/techies_remote_mines.txt [hunk 1]
```diff
@@ -22,7 +22,7 @@
     // Casting
     //-------------------------------------------------------------------------------------------------------------
     "AbilityCastRange"                                    "500"
-    "AbilityCastPoint"                                    "1.0"
+    "AbilityCastPoint"                                    "0.72" //OAA
```
Comment (POCKETMONEY99) [inline game/scripts/npc/abilities/techies_remote_mines.txt:25]: I would leave this unchanged. This is something which could change the feel for techies mains.

File: game/scripts/npc/items/custom/item_trumps_fists.txt [hunk 2]
```diff
@@ -72,22 +72,22 @@
...
       {
         "var_type"                                        "FIELD_INTEGER"
-        "heal_prevent_percent"                            "-50 -75"
+        "heal_prevent_percent"                            "-75"
       }
       "06"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "heal_prevent_duration"                           "3"
+        "heal_prevent_duration"                           "1 2"
```
Comment (POCKETMONEY99) [inline game/scripts/npc/items/custom/item_trumps_fists.txt:90]: Leave the heal prevention duration at 3 seconds.

File: game/scripts/npc/abilities/chaos_knight_phantasm.txt [hunk 3]
```diff
@@ -41,7 +41,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "images_count"                                    "3"
+        "images_count"                                    "4" //OAA
```
Comment (POCKETMONEY99) [inline game/scripts/npc/abilities/chaos_knight_phantasm.txt:44]: Remove all of your CK changes. Let Darkonius CK changes go through first the see how the hero is.

File: game/scripts/npc/abilities/ancient_apparition_ice_blast.txt [hunk 4]
```diff
@@ -63,7 +63,7 @@
       "05"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "frostbite_duration"                              "10 11 12 13 14"
+        "frostbite_duration"                              "10 11 12 14 16" //OAA
```
Comment (POCKETMONEY99) [inline game/scripts/npc/abilities/ancient_apparition_ice_blast.txt:66]: Revert this change. AA's ultimate lasts long enough for it to be effective. Can you instead reduce the mana cost on chilling touch to something reasonable at OAA levels.

File: game/scripts/npc/abilities/furion_wrath_of_nature_oaa.txt [hunk 5]
```diff
@@ -74,7 +74,7 @@
       "07"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "kill_damage"                                     "4 5 6 7 8"
+        "kill_damage"                                     "6 10 14 20 30" //OAA
```
Comment (POCKETMONEY99) [inline game/scripts/npc/abilities/furion_wrath_of_nature_oaa.txt:77]: Can you give this whole spell reasonable scaling. The numbers are all over the place.

File: game/scripts/npc/abilities/furion_wrath_of_nature_oaa.txt [hunk 6]
```diff
@@ -52,7 +52,7 @@
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage_percent_add"                              "11"
+        "damage_percent_add"                              "12" //OAA
```
Comment (POCKETMONEY99) [inline game/scripts/npc/abilities/furion_wrath_of_nature_oaa.txt:55]: Leave the percentage unchanged. Even a 1% increase in the damge of this ultimate is huge. You should increase the base damage of this spell.

File: game/scripts/npc/abilities/techies_remote_mines.txt (hunk 1 above)
Comment (lovesmuffins) [reply game/scripts/npc/abilities/techies_remote_mines.txt:25]: I have tested this out myself and .72 is a buff whilst still feeling like old techies. I will wait for other feedback as the purpose of the buff and what I've been suggested to do is to decrease castpoint.

File: game/scripts/npc/items/custom/item_trumps_fists.txt (hunk 2 above)
Comment (lovesmuffins) [reply game/scripts/npc/items/custom/item_trumps_fists.txt:90]: I might make it 2 3 (since status resistance)

either way the code is bugged

File: game/scripts/npc/abilities/chaos_knight_phantasm.txt (hunk 3 above)
Comment (lovesmuffins) [reply game/scripts/npc/abilities/chaos_knight_phantasm.txt:44]: i will talk to darkonius

File: game/scripts/npc/abilities/ancient_apparition_ice_blast.txt (hunk 4 above)
Comment (lovesmuffins) [reply game/scripts/npc/abilities/ancient_apparition_ice_blast.txt:66]: this is a needed change to AA. 

As baumi once said "if u want to buff a hero, buff it's strengths"

xd

File: game/scripts/npc/abilities/furion_wrath_of_nature_oaa.txt (hunk 5 above)
Comment (lovesmuffins) [reply game/scripts/npc/abilities/furion_wrath_of_nature_oaa.txt:77]: first of all, not buffing the damage...

kill damage is what chris, max, and I came up with. We leave it at these values for now please :)

Comment (Larenall) [reply game/scripts/npc/abilities/furion_wrath_of_nature_oaa.txt:77]: it doesnt matter how high kill dmg is if you cant kill a single creep...

Comment (lovesmuffins) [reply game/scripts/npc/abilities/furion_wrath_of_nature_oaa.txt:77]: > it doesnt matter how high kill dmg is if you cant kill a single creep...

darkonius is working on adding to the lua. It will insta kill creeps (not ancients, dragons, or bosses) and these changes will go along with those changes

Comment (lovesmuffins) [reply game/scripts/npc/abilities/furion_wrath_of_nature_oaa.txt:77]: update on ^. The numbers will be toned down obviously.

**Comment by DarkoniusXNG** [comment]
I will close this. You make a new PR with just techies changes. We discussed other stuff.
